packages/Card/src/test.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import Banner from './Banner';
+
+export default { title: 'Example/Banner', component: Banner };
+export const Info = () => (
+  <Banner variant="info">
+    {' '}
+    <p>
+      {' '}
+      This is an example of an info banner to display important information.{' '}
+    </p>{' '}
+  </Banner>
+);
+export const Danger = () => (
+  <Banner variant="danger">
+    {' '}
+    <p>This is an example of a danger banner to display warnings.</p>{' '}
+  </Banner>
+);
+export const Congrats = () => (
+  <Banner variant="congrats">
+    {' '}
+    <p>This is an example a congrats banner to celebrate a win!</p>{' '}
```

Comment:
Pas très grave mais il manque un "of" après "example" dans le texte, pareil pour celui en bas.

Reply:
En fait, ce fichier ne devrait pas exister c'est une erreur.